public/app/features/dashboard/partials/dashboard_import.html

```diff
@@ -80,6 +80,20 @@ <h3 class="section-heading">
         </div>
       </div>
 
+      <div class="gf-form-inline">
```

Comment:
Would like to display folder selection before dashboard title input instead of after - feels more logical. What do you think?

Reply:
Yes, that's a good point. So disregard this.